Makefile

```diff
@@ -110,6 +110,7 @@ check-go:
 ## ccheck-no-changes : Check there is no local changes.
 .PHONY: check-no-changes
 check-no-changes:
+	@go mod tidy
```

Comment:
If we run `go mod tidy` here we need to run also `go mod download` to be consistent with `make notice`. I have opened a PR for this: https://github.com/elastic/beats/pull/26241